StRoot/StFttHitCalibMaker/StFttHitCalibMaker.cxx

```diff
@@ -49,9 +51,31 @@ Int_t StFttHitCalibMaker::FinishRun(Int_t runnumber)
 Int_t StFttHitCalibMaker::Finish()
 { 
     LOG_INFO << "StFttHitCalibMaker::Finish()" << endm;
+
+    if (this->mCalibMode == StFttHitCalibMaker::CalibMode::Calibration) {
+        LOG_INFO << "Writing StFttHitCalib parameters to plaintext: " << endm;
+        // this->WriteCalibrationToPlainText();
+    }
+    this->WriteCalibrationToPlainText();
```

Comment:
This needs a confirmation that `WriteCalibrationToPlainText()` should be called regardless of `mCalibMode` value. If it is so, the log message is not relevant.

Also, we don't use `this->` for class members.

Reply:
Fixed in 28da7017d2f9ed86b953205c21800711d2968e3e